extension/src/plots/vega/util.test.ts

```diff
@@ -275,7 +278,7 @@ describe('extendVegaSpec', () => {
   it('should update the multi-source template to remove erroneous shape encoding from the vertical line displayed on hover', () => {
     const updatedSpec = extendVegaSpec(
       multiSourceTemplate,
-      PlotSizeNumber.LARGE,
+      PlotNumberOfItemsPerRow.LARGE,
```

Comment:
[C] Should probably change the names in the enum from LARGE etc to the numbers.

Reply:
I'm doing a follow up. I first thought of adding `height` inside of `plotSizes`, but it'll be way simpler to just add another property. I'll rename these at the same time.